package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "snowsql-formatter",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
did you mean for this to match what's in `package-lock.json`?

Reply:
Oh meant to delete that file. Do you think it makes sense to include that in the repo? This [doc](https://github.com/npm/cli/blob/release-6.14.7/docs/content/configuring-npm/package-lock-json.md) seems to suggest that we should push it to version control.

Reply:
I think so -- that "integrity" piece is what allows us to have reproducible builds, i.e. making sure that we have an audit trail of which code was included at which point

Reply:
Alright. Will include it with the correct version.